datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -288,6 +287,25 @@ def create_scraper_configuration(self, instance=None):
         if config['metadata_metric_name'] and config['metadata_label_map']:
             config['_default_metric_transformers'][config['metadata_metric_name']] = self.transform_metadata
 
+        # Set up the HTTP wrapper for this endpoint
+
+        # TODO: Deprecate this behavior in Agent 8
+        if config['ssl_verify'] is False:
+            config.setdefault('tls_ignore_warning', True)
+
+        http_handler = self.http_handlers[endpoint] = RequestsWrapper(
+            config, self.init_config, self.HTTP_CONFIG_REMAPPER, self.log
+        )
+
+        headers = http_handler.options['headers']
+
+        bearer_token = config['_bearer_token']
+        if bearer_token is not None:
+            headers['Authorization'] = 'Bearer {}'.format(bearer_token)
+
+        # TODO: Determine if we really need this
+        headers.setdefault('accept-encoding', 'gzip')
```

Comment:
`headers` and `http_handler` are never assigned to anything?

Reply:
?

Reply:
Oh, I see you are doing double assignment a few lines up:
```
        http_handler = self.http_handlers[endpoint] = RequestsWrapper(
             config, self.init_config, self.HTTP_CONFIG_REMAPPER, self.log
         )
```

Agree with Alex's comment, this should really be in a separate function since it doesn't make changes to the `config` object at all

Reply:
It does on L294

Reply:
While that's true - from a readability standpoint, there are over two hundred lines with assignments to `config`, so this last set of stanzas seems out of place.  The double assignment above was easily overlooked too leading to my confusion.